src/process/tests.rs

```diff
@@ -84,6 +87,28 @@ async fn merge_blocked_by_label() {
     );
 }
 
+#[tokio::test]
+async fn merge_not_blocking_on_reviews() {
+    let (mut pr, client, mut config) = make_context();
+
+    // It's trivial
+    config.trivial_review_label = Some("trivial :)".to_string());
+    pr.labels.insert("trivial :)".to_string());
+
+    let mut analyzer = make_analyzer(&pr, &client, &config);
+    // But also someone requested changes
+    analyzer.reviews = RemoteData::Local(vec![review("1", ReviewState::ChangesRequested)]);
+
+    assert_eq!(
+        analyzer.required_actions().await.unwrap(),
+        *Actions::noop()
+            .set_merge(true)
+            .set_label("reviewed", Presence::Absent)
+            .set_label("ci-passed", Presence::Present)
+            .set_label("needs-description", Presence::Absent)
+    );
```

Comment:
Does this mean if someone requests changes on a trivial pr it will still be merged. That doesn't feel right to me 🤔

Reply:
Ah yeah, we could make it so that if somebody had the time to make a review, then the PR would not get merged 👍